.regen

```diff
@@ -1 +1 @@
-3
\ No newline at end of file
+4
```

Comment:
@bkeese Perhaps add a non-sensical suffix (`4blerg`).  Perhaps CircleCI has a regen 4 cached.  I know that in the main branch we're up to 32.